.github/workflows/publish-to-docker.yml

```diff
@@ -32,7 +36,7 @@ jobs:
         id: version
         run: |
           if [ "${{ github.event_name }}" = "push" ]; then
-            echo "VERSION=0.0.63.dev20250114" >> $GITHUB_OUTPUT
+            echo "VERSION=0.0.63.dev51206766" >> $GITHUB_OUTPUT
```

Comment:
what is the significance of this placeholder?

Reply:
This line is reached when we trigger a workflow on git push, which is how I am triggering a test workflow to test the PR. The version is a TestPyPi package built on latest main.